scripts/extract_mecas.sh

```diff
@@ -67,20 +67,19 @@ for file in $INCOMING_DIR/*; do
 
     echo "$uuid" >"$outputDir/source.txt"
 
-    if [ "$doiPrefix" = "10.1101" ]; then
-        pull_docker_image
-        echo "correct some bioRxiv/Encoda XML issues and store $outputDir/$id.xml..."
-        cat "$tmpDir/$xmlFile" | docker run --rm -i ghcr.io/elifesciences/enhanced-preprints-biorxiv-xslt:latest /app/scripts/transform.sh --doi $doiSuffix >"$outputDir/$id.xml"
-        echo "transform.sh --doi $doiSuffix successfully run"
-    else
-        echo "Skipping XML correction for DOI prefix $doiPrefix"
-        cp "$tmpDir/$xmlFile" "$outputDir/$id.xml"
-    fi
-
-    echo "copy all tif, gif and jpg content to ${outputDir}..."
-    cp $tmpDir/content/*.tif "$outputDir/" || true
-    cp $tmpDir/content/*.gif "$outputDir/" || true
-    cp $tmpDir/content/*.jpg "$outputDir/" || true
+    pull_docker_image
```

Comment:
If we are using the xsl to account for changes that are required to encoda or the EPP platform, (i.e. issues that are not caused by XML), then we need the xsl to be run on all content.

For example, if an article has an appendix, and encoda doesn't currently extract this, then we need the xsl that converts appendices to sections to be run on content that is also not from bioRxiv.

The alternative would be me having to keep track of all these issues, and to request that Exeter provide broken/incorrect XML so that we can render the content.

Reply:
Would be happy to discuss this through if this is unclear.